zson/marshal_test.go

```diff
@@ -235,3 +235,50 @@ func TestUnexported(t *testing.T) {
 	_, err := m.Marshal(f)
 	require.NoError(t, err)
 }
+
+type EmbeddedZNG struct {
```

Comment:
I would think an embedded test would be:

```
type Bar struct {
    V1 string `zng:"v1"`
    V2 string `zng:"v2"`
}

type Foo struct {
    Type string `zng:"type"`
    Bar
}
```

Which the marshal to:

```
{
    "type": "Bar",
    "v1": "v1",
    "v2":"v2",
}
```

Reply:
Are you asking me to change the word "embedded" in the test?  I am using it to mean you can embed zng.Values in zng-marshaled Go structs.    We don't hoist structs like your example and neither does encoding/json.  Mapstructure has an option to do this.  It feels low priority.

Reply:
@mattnibs point might be that "embedded" in the Go spec denotes something different than what's going on here. For that reason, I'd encourage use of a different term like "zng.Value field".